debezium-core/src/main/java/io/debezium/time/Conversions.java

```diff
@@ -133,6 +133,16 @@ protected static LocalDateTime toLocalDateTime(Object obj) {
             LocalTime localTime = toLocalTime(obj);
             return LocalDateTime.of(EPOCH, localTime);
         }
+        if ( obj instanceof java.sql.Timestamp) {
+            java.sql.Timestamp timestamp = (java.sql.Timestamp)obj;
+            return LocalDateTime.of(timestamp.getYear() + 1900,
```

Comment:
Would be nice to use the non-deprecated replacements of these methods, but I don't mind to much. Going to merge it, feel free to send a follow-up if you like.